src/CarlaCDASimAdapter.py

```diff
@@ -140,11 +143,23 @@ def __get_detected_objects(self, infrastructure_id, sensor_id):
         default="INFO",
         type=str,
         help="Log Level for service (default: INFO)")
+    arg_parser.add_argument(
+        "--log-file-path",
+        default="/home/CarlaCDASimAdapter/logs/",
+        type=str,
+        help="Absolute path for log file.")
     
     args = arg_parser.parse_args()
     log_level = logging.getLevelName(args.log_level)
+    log_file_name = datetime.datetime.now().strftime("carla_cdasim_adapter_%H_%M_%d_%m_%Y.log")
     FORMAT = '%(asctime)s:%(levelname)s:%(message)s'
-    logging.basicConfig(format=FORMAT, filename='cdasim_adapter.log', level=log_level)
+    logging.basicConfig(format=FORMAT,  
```

Comment:
`FORMAT` is only ever used once, so we should replace
```python
format=FORMAT
```
with 
```python
format='%(asctime)s:%(levelname)s:%(message)s'
```

Reply:
No, This is easier to read

Reply:
I disagree. One of the benefits of named arguments is this exact situation. However, we can keep it as is since that code was already there.